c2corg_api/tests/views/test_user_mailinglists.py

```diff
@@ -49,8 +49,7 @@ def test_post_mailinglists_unauthenticated(self):
     def test_post_mailinglists_invalid(self):
         request_body = {
             'wrong_mailinglist_name': True,
-            'avalanche': 'incorrect_value',
-            'avalanche': False
```

Comment:
I think duplicate entries are not really a problem, because you are sending a JSON object, which can not contain multiple values with the same key. :)